src/test/java/wooteco/subway/web/member/LoginMemberControllerTest.java

```diff
@@ -0,0 +1,158 @@
+import org.springframework.restdocs.RestDocumentationExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.filter.ShallowEtagHeaderFilter;
+import wooteco.subway.doc.MemberDocumentation;
+import wooteco.subway.domain.member.Favorite;
+import wooteco.subway.domain.member.Member;
+import wooteco.subway.domain.member.MemberRepository;
+import wooteco.subway.infra.JwtTokenProvider;
+import wooteco.subway.service.favorite.dto.FavoriteResponse;
+import wooteco.subway.service.member.MemberService;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.when;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.documentationConfiguration;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static wooteco.subway.AcceptanceTest.*;
+
+@ExtendWith(RestDocumentationExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+class LoginMemberControllerTest {
+
+    @MockBean
+    protected MemberService memberService;
+
+    @MockBean
+    protected MemberRepository memberRepository;
+
+    @Autowired
+    protected JwtTokenProvider jwtTokenProvider;
+
+    protected MockMvc mockMvc;
+    private Member member;
+    private String token;
+    private List<Favorite> favorites;
+
+    @BeforeEach
+    public void setUp(WebApplicationContext webApplicationContext, RestDocumentationContextProvider restDocumentation) {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext)
+                .addFilter(new ShallowEtagHeaderFilter())
+                .apply(documentationConfiguration(restDocumentation))
+                .build();
+
+        favorites = Arrays.asList(new Favorite(1L, 1L, 2L), new Favorite(2L, 2L, 3L), new Favorite(3L, 3L, 4L));
+        member = new Member(TEST_USER_EMAIL, TEST_USER_NAME, TEST_USER_PASSWORD, new HashSet<>(favorites));
+        token = jwtTokenProvider.createToken(member.getEmail());
+    }
+
+    @Test
+    void login() throws Exception {
+        given(memberService.createMember(any())).willReturn(member);
+
+        String inputJson = "{\"email\":\"" + TEST_USER_EMAIL + "\"," +
+                "\"name\":\"" + TEST_USER_NAME + "\"," +
+                "\"password\":\"" + TEST_USER_PASSWORD + "\"}";
+
+        this.mockMvc.perform(post("/oauth/token")
+                .header("Authorization", "bearer " + token)
+                .content(inputJson)
+                .accept(MediaType.APPLICATION_JSON_UTF8)
```

Comment:
```suggestion
                .accept(MediaType.APPLICATION_JSON)
```

https://bugs.chromium.org/p/chromium/issues/detail?id=438464 를 참고해주세요~

Reply:
수정하였습니다. 위 글을 보았을 때, 결국 무시된다는 것으로 이해했는데 맞는것인가요?

Reply:
네 맞습니다! 인코딩이 포함되는 것은 표준이 아니고 브라우저 마다 다르게 처리되고 있는 상황입니다.
인코딩을 포함하는 것이 잘못된 방법이라는 것을 나타내고 있네요!